test/functional/terminal/channel_spec.lua

```diff
@@ -92,3 +93,18 @@ describe('terminal channel is closed and later released if', function()
     eq(chans - 1, eval('len(nvim_list_chans())'))
   end)
 end)
+
+if iswin() then
```

Comment:
not really any reason for this test to only run on Windows. We just would only use `sh` for non-Windows systems.

Reply:
Done